registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -216,6 +217,7 @@
     "sha256:47e89f01c17f66513adddac0b48e035ab937469f3b93df7ab704d725d1e611c1": ["v0.2.0"]
 - name: clusterctl
   dmap:
+    "sha256:b3927e1b34d7df7b27ea0591fe40706620c5fde0df65a8f50ba4ad272c1587f5": ["v1.10.0"]
```

Comment:
`b3927e1b34d7` matches the SHA for clusterctl v1.10.0 in the [staging repository](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/clusterctl@sha256:b3927e1b34d7df7b27ea0591fe40706620c5fde0df65a8f50ba4ad272c1587f5/details) 👍🏻